sql/src/main/java/org/apache/druid/sql/calcite/rel/DruidQuery.java

```diff
@@ -229,6 +229,15 @@ public static DruidQuery fromPartialQuery(
               virtualColumnRegistry
           )
       );
+    } else if (partialQuery.getUnnestFilter() != null) {
+      filter = Preconditions.checkNotNull(
+          computeUnnestFilter(
+              partialQuery,
+              plannerContext,
+              sourceRowSignature,
+              virtualColumnRegistry
+          )
+      );
```

Comment:
can you not have both a where filter an an unnest filter? is an unnest filter basically a where filter that got picked up somewhere else?

Reply:
Yes an unnest filter is same as a where or having that gets picked up somewhere else